src/Components/Auth/Login.tsx

```diff
@@ -277,10 +277,7 @@ export const Login = (props: { forgot?: boolean }) => {
             <div className="relative flex h-full w-full items-center">
               <div
                 className={
-                  "w-full transition-all " +
-                  (!forgotPassword
-                    ? "visible -translate-x-0 opacity-100"
-                    : "invisible -translate-x-5 opacity-0")
+                  "w-full transition-all " + (!forgotPassword ? "" : "hidden")
```

Comment:
instead of string concatenation, use the classNames utility which makes it easy to format and read
```suggestion
                  classNames("w-full transition-all", forgotPassword && "hidden")
```